### ISSUE TITLE
constant app badge

### ISSUE THREAD

**Issue by dschneiderch** [wavebox/waveboxapp#472]
* Which version of Wavebox are you using? 3.5.0

* Which Operating System are you using? macos 10.12.6

I'm seeing an app badge "2" for one of my 2 gmail accounts but there are no new unread messages. I've tried toggling "show unread count in sidebar or toolbar" and restarting but the 2 keeps coming back. I currently have "Unread & Sync" setup for "Unread Messages".  I also tried shift+esc a couple times based on a closed issue but that didn't help.
any suggestions?
Thanks

**Comment by nmat** [comment]
Hello @dschneiderch, did you follow: https://wavebox.io/kb/google-unread-counts?

not 100% sure what you mean here.

**Comment by dschneiderch** [comment]
I had not but i'm set up correctly.
Inbox type: Default
WBox: Unread in Primary Category

fyi, I am using multiple inboxes in gmail. but there still aren't any unread messages even though the badge says 2

**Comment by dschneiderch** [comment]
there are no unread messages when I perform that search

**Comment by Thomas101** [comment]
We have seen a sporadic bug with Google Sync that leaves a couple of unread messages marked as so. If you try re-adding the account does that still sit at two?

**Comment by dschneiderch** [comment]
re-adding the accounts did the trick.

I also learned that toggling the Sidebar Badge to show unread count from all services was causing the badge to be hidden on my other gmail account. This seems to be misleading. I don't have any other services since I'm on the free version but if I did I wouldn't want these services to be included in the badge count. in my opinion, the description does not imply that this option would affect the unread badge for your **email**.
